inspirehep/modules/authors/utils.py

```diff
@@ -313,5 +312,6 @@ def __check_parts(parts):
 
 def author_tokenize(phrase):
     """Return all possible variatons of a name"""
-    return parse_scanned_author_for_phrases(
+    name_variations = parse_scanned_author_for_phrases(
         scan_author_string_for_phrases(phrase))
+    return [x.lower() for x in name_variations]
```

Comment:
Mmh. I thought we decided we let ES to lower everything, directly in the mapping configuration. Or I am wrong? @jmartinm